From fae9f462e31c935c973850bae1d38d5fe69a082c Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Tue, 31 Jul 2018 17:52:11 -0400
Subject: [PATCH] TRITON-653 vm-agent TRITON-628 bug - missing jsprim
 dependency Reviewed by: Rui <rui@joyent.com> Approved by: Rui
 <rui@joyent.com>

---
 package.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package.json b/package.json
index c60ac42..cc9fcad 100644
--- a/package.json
+++ b/package.json
@@ -10,6 +10,7 @@
         "cueball": "2.1.1",
         "deep-diff": "0.3.3",
         "eslint": "2.13.1",
+        "jsprim": "2.0.0",
         "kthxbai": "0.4.0",
         "lstream": "0.0.4",
         "mockery": "1.4.0",
-- 
2.21.0

